docs/tutorials/patterns.md

```diff
@@ -283,3 +283,84 @@ Webhook GitHub:
 Finally, the GitHub WebHook payload url is the url above with `<secret>`
 replaced with base64 decoded string of the value of `data.WebHookSecretKey`
 above and the content type is `application/json`.
+
+## Custom domain names and secure transport
+
+Custom domain names and HTTPS secure data transport are implemented in the
+Route object level. They are controlled with the keywords `spec.host` and
+`spec.tls`.
+
+The public DNS entry of the custom domain name should point to public IP of the
+domain `rahtiapp.fi` (currently `193.167.189.101`) and the custom DNS name is
```

Comment:
I think it would be better to have the custom domain name to refer to 'rahtiapp.fi' instead of the IP, which might actually change.